src/archivematicaCommon/lib/archivesspace/client.py

```diff
@@ -130,6 +130,17 @@ def resource_type(self, resource_id):
     def get_record(self, record_id):
         return self._get(record_id).json()
 
+    def get_levels_of_description(self):
+        """
+        Returns an array of all levels of description defined in this ArchivesSpace instance.
+        """
+        if not hasattr(self, 'levels_of_description'):
+            # TODO: * fetch human-formatted strings
+            #       * is hardcoding this ID okay?
```

Comment:
Haven't heard back yet; I think it may take a bit to get a response. Maybe merge now and update later with the info we get?